packages/static-wado-scp/lib/program/index.js

```diff
@@ -10,7 +10,7 @@ function main() {
     console.log("Network error: ", e);
   });
   console.log(`Starting server listen on port ${port}`);
-  server.listen(port);
+  server.listen(port, this.dicomWebScpConfig);
```

Comment:
@wayfarer3130  , this is regarding the issue we discussed on call. 
This is indeed a natural solution to me, what do you think?

Reply:
These are the ae settings that are getting applied, so I think it should be:
const aeData = aeConfig[this.dicomWebScpConfig.staticWadoAe];
const server = new Server(port, aeData)
...
server.listen(port,aeData)
because the settings are for the ae, not for the program being run.

Reply:
> because the settings are for the ae, not for the program being run.
Agree on that.

But the point is server.listen(port,options), the options will be used to pass down to DcmjsDimseScp constructor.
What we really need here is the running configs such as:
isStudy, isGroup, maximumInlinePrivateLength, maximumInlinePublicLength.. So aeData will not contain the DcmjsDimseScp opts we need..

Another alternative solution is to decouple importer initialization from DcmjsDimseScp to a secondary step. 
It would be something like this
```

class DcmjsDimseScp extends Scp {
  constructor(socket, opts = {}) {
    super(socket, opts);
    this.association = undefined;
    this.options = opts;
  }
  
  initImporter(params) {
      this.importer = new StaticWado(opts);
  }
```
and use it as following
scp/lib/program/index.js
```

server.listen(port);
server.clients.forEach(client => if(client instanceof DcmjsDimseScp) {client.initImporter(this.dicomWebScpConfig)}
```